### PR TITLE
Solved eslint errors for beer-song

### PR DESCRIPTION
**Pull request by Futuro212**
Fixed ESLint errors for the beer-song exercise (for issue #480)

### CODE REVIEW

**Comment by tejasbubane** [comment]
@Futuro212 Please resolve merge conflicts here. Changes look good.

**Comment by tejasbubane** [comment]
Sorry there's conflict again because of merging `binary-search` PR.

File: exercises/beer-song/example.js [hunk 1]
```diff
@@ -19,14 +19,14 @@ function action(currentVerse) {
...
-}
-
 function nextVerse(currentVerse) {
   return currentVerse === 0 ? 99 : currentVerse - 1;
 }
 
+function nextBottle(currentVerse) {
+  return `${bottles(nextVerse(currentVerse)).toLowerCase()} of beer on the wall.\n`;
+}
+
```
Comment (tejasbubane) [inline exercises/beer-song/example.js]: Moving this function is not required I think.

**Comment by Futuro212** [comment]
Fixed conflict

File: exercises/beer-song/example.js (hunk 1 above)
Comment (Futuro212) [reply exercises/beer-song/example.js]: It is because the nextVerse function needs to be defined before the nextBottle one.

Comment (tejasbubane) [reply exercises/beer-song/example.js]: Was that an eslint error?

Comment (Futuro212) [reply exercises/beer-song/example.js]: `23:21  error  'nextVerse' was used before it was defined  no-use-before-define`

Something like this.

Comment (tejasbubane) [reply exercises/beer-song/example.js]: Ahh ok 👍

**Pull request merged**